src/pages/LandingPages/MGCovid19/MGCovidHero.vue

```diff
@@ -84,6 +84,10 @@ export default {
 		},
 		result({ data }) {
 			this.covidLandingActive = _get(data, 'general.mg_covid_active.value') === 'true' || false;
+			// Force the covid landing form to show when running the GROW-79 experiment
+			if (!this.covidLandingActive && this.isExperimentActive) {
```

Comment:
We should not be overriding this feature flag.

It's purpose is two-fold; first as a gate in production today which you don't have to worry about and second as a kill switch for preventing entry into MG using the Covid Category and/or using the onetime deposit option.

Should there be a need to turn this off quickly we want to have a way that isn't subject to stale cookies in the browser as an experiment might be. That said, we would most likely want to turn off the experiment too.

Reply:
Got it. Thanks!